packages/subgraph/tests/plugin/adminMembers.test.ts

```diff
@@ -37,14 +36,20 @@ describe('AdminMembers', function () {
   });
 
   test('handleGranted', function () {
+    // check the entities are not in the store
+    assert.entityCount('Administrator', 0);
+    assert.entityCount('AdministratorAdminPlugin', 0);
+
+    // create the event and handle it
     let event = createGrantedEvent(
+      EXECUTE_PROPOSAL_PERMISSION_HASH,
       DAO_ADDRESS,
       pluginEntityId,
-      adminEntityId,
-      AdminPermission
+      adminEntityId
     );
     handleGranted(event);
 
+    // check the administrator entity
     assert.entityCount('Administrator', 1);
     assert.fieldEquals('Administrator', adminEntityId, 'id', adminEntityId);
```

Comment:
perhaps `administratorEntityId` is a better naming instead of `adminEntityId`, WDYT?